MalwareBuilder.py

```
@@ -1,58 +1,62 @@
+class Construct:
     @classmethod
-    def exe(cls, script_names: list[str]):
+    def exe(cls, script_names: list[str], config_path: str = "config.ini"):
+        config = configparser.ConfigParser()
+        config.optionxform = str  # Preserve original case of keys
+        config.read(config_path)
+
         # Uninstall pathlib to avoid conflicts with PyInstaller
         REINSTALL = False
         if importlib.util.find_spec('pathlib') is not None:
             log.info(f"[-] Uninstalling pathlib to avoid conflicts with PyInstaller")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)